src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -18,8 +19,7 @@ object AstTools {
     def findTerminals(): Seq[Terminal] = AstTools.findTerminals(astNode)
     def findTopLevelMemberAccesses(): Iterable[Ast] = AstTools.findTopLevelMemberAccesses(astNode)
     def sourceString(): String = astNode.asInstanceOf[Terminal].getSourceString
-    
-    def wdlType: WdlType = {
+    def wdlType(wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): WdlType = {
```

Comment:
At this point I think we should set up a ticket to go in and figure out how to do this better all across the board, but for now I think it's out of scope of these sorts of changes